client/src/main/java/org/apache/iotdb/client/AbstractClient.java

```diff
@@ -90,7 +90,7 @@
   private static final String DEFAULT_TIME_FORMAT = "default";
   private static String timeFormat = DEFAULT_TIME_FORMAT;
   static int maxPrintRowCount = 1000;
-  private static int fetchSize = 10000;
+  private static int fetchSize = maxPrintRowCount;
```

Comment:
Excuse me, do you know the difference between `fetchSize` and `maxPrintRowCount`?
Have you noticed the line 781 `statement.setFetchSize(fetchSize);` ?
Your change will make the client try to fetch `maxPrintRowCount` rows every time.
What if I set `maxPrintCount` to be 1000_1000?

Reply:
i think it is init number not dynamic number，print once and fetch once

Reply:
> @jixuan1989 why ci test has error ? it looks like ci bug..
> 
> > https://travis-ci.org/apache/incubator-iotdb/builds/607035699?utm_source=github_status&utm_medium=notification

let me have a look..

Reply:
Actually I agree with LeiRui here, as the default values are different...

Reply:
if server have 10000 line . we just print 1000.

Reply:
maybe set fetchsize=1000 and printsize=1000 is true，not set fetchsize=printsize ！ and you？

Reply:
Yes, I agree that the old parameter setting is not so good.
`maxPrintRowCount` should not be less than `fetchSize`.
Maybe keep `fetchSize` as the original value 10_000 while assign `maxPrintRowCount` to be 100_000.

Reply:
👌，i will fix it